query/search.js

```diff
@@ -37,7 +37,7 @@ query.score( peliasQuery.view.address('postcode') );
 // multi_match is appropriate.
 query.score( peliasQuery.view.admin('country_a') );
 query.score( peliasQuery.view.admin('region_a') );
-query.score( peliasQuery.view.admin_multi_match(adminFields), 'peliasAdmin' );
+query.score( peliasQuery.view.admin_multi_match(adminFields, 'peliasAdmin') );
```

Comment:
I included this fix which is now needed because we fixed the admin_multi_match view. Thanks to @trescube for finding this!